qt/scientific_interfaces/Muon/MuonAnalysis.cpp

```diff
@@ -1826,7 +1826,7 @@ void MuonAnalysis::plotSpectrum(const QString &wsName, bool logScale) {
   s << "def plot_data(ws_name,errors, connect, window_to_use):";
   if (parsePlotType(m_uiForm.frontPlotFuncs) == PlotType::Asymmetry) {
     // clang-format off
-    s << "  w = plotSpectrum(source = ws_name,"
+    s << "  w = pymantidplot.plotSpectrum(source = ws_name,"
```

Comment:
I was wondering if we could fix this another way with this patch: https://gist.github.com/martyngigg/0db2f49bb1d70fbdd85a8435a69e59f2. It will fix all of the plotting I think as it will just delete the new `plotSpectrum` from `mantid.simpleapi`.

Reply:
Then we won't need to call `pymantidplot` especially anywhere.

Reply:
Yeah, that seems to work well and is a nice solution to the issue :+1:  . Maybe I don't have a full understanding of the problem, but would we never want to call the new versions of plotSpectrum and plotBin from MantidPlot, or are they inherently incompatible?

Reply:
The main problem was that any import like `from mantid.simpleapi *` would just overwrite the MantidPlot ones and any call to it in MantidPlot should use the old API.  They are supplied for workbench as a transition to keep scripts working and if they need to be accessed in workbench it would be possible to do from `mantid.plots import plotSpectrum` still.